gammapy/utils/units.py

```diff
@@ -0,0 +1,44 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Units and Quantity related helper functions"""
+from __future__ import absolute_import, division, print_function, unicode_literals
+import astropy.units as u
+
+__all__ = [
+    'standardise_unit',
```

Comment:
@mhvk may disagree, but I was just suggesting using the equivalency as a way to "standardise the unit" when you read, without having to go digging through the bases. For example:

```python
q1 = 10 * u.photon / u.cm**2 / u.s
q2 = 10 * u.count / u.cm**2 / u.s

def standardise_unit(q):
    eqs = [(u.Unit("ph/(cm2 s)"), u.Unit("1/(cm2 s)")),
           (u.Unit("ct/(cm2 s)"), u.Unit("1/(cm2 s)"))]
    return q.to("cm-2 s-1", equivalencies=eqs)

standardise_unit(q1)
```